msteams-platform/feedback.md

```diff
@@ -15,6 +15,7 @@ Microsoft Teams community of developers use Stack Overflow to connect with other
 |         **Community chat group**         |  Select **SIGN IN TO START TALKING** to join the informal community chat room for [Microsoft Teams app developers](https://gitter.im/OfficeDev/MicrosoftTeamsAppDev), if you are signing in for the first time. Select **JOIN ROOM** if you are already logged in.      |
 |            **Community support**             |     <ul><li> [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams): Use the `botframework & microsoft-teams` tag to post the questions. You must follow the Stack Overflow guidelines, such as provide a descriptive title, a complete and concise problem statement, along with sufficient detail to reproduce your issue. Feature requests or broad questions are off-topic. If you are a new user, for more information, see Stack Overflow Help Center. </li>                                                                                                                                                                       <li>  [Microsoft Q&A](/answers/topics/office-teams-app-dev.html): Use the `office-teams-app-dev` tag to post questions. We love Stack Overflow and our extensive support is assured for customers who ask questions there. However, Stack Overflow has specific criteria about questions that are appropriate for the community and [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) has an open policy regarding this.  </li> </ul>                                                                                            |
+|          **Teams Toolkit or TeamsFx SDK and CLI issues**           |     <ul><li> [Github Issues](https://github.com/OfficeDev/TeamsFx/issues) Create new issue on the [TeamsFx (Microsoft Teams Framework) GitHub repository](https://github.com/OfficeDev/TeamsFx) to report issues or raise feature request. We love Github Issues and our extensive support is assured for customers who ask questions there. </li>                                                                                                                                                                       <li>  [Stack Overflow](https://stackoverflow.com/questions/tagged/teams-toolkit): Use the `teams-toolkit` tag to post questions. You must follow the Stack Overflow guidelines, such as provide a descriptive title, a complete and concise problem statement, along with sufficient detail to reproduce your issue. Feature requests or broad questions are off-topic. If you are a new user, for more information, see Stack Overflow Help Center.  </li> </ul>                                                                                            |
```

Comment:
Please rephrase:
We love Github Issues and our extensive support is assured for customers who ask questions there.

Reply:
done rephrasing, please check.